doc/api.md

```diff
@@ -1,3 +1,12 @@
+$ /Users/danielgrosse/Projects/plugins/plugin-client-sdk/node_modules/.bin/jsdoc2md src/main.js
```

Comment:
```suggestion
```
your personal path somehow found its way here

Reply:
Thanks I fixed the root cause